tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -374,7 +374,16 @@ module.exports = {
         .waitForElementNotPresent('@loadingIndicator')
         .waitForElementPresent(this.getFileRowSelectorByFileName(element))
     },
-
+    assertSharingIsDisabled: function () {
+      this.assert.elementNotPresent('shareButtonInFileRow')
+      return this
+        .waitForElementVisible('@firstFileRow')
+        .useXpath()
+        .click('@firstFileRow')
+        .assert
+        .elementNotPresent('//div[@class="uk-active"]')
```

Comment:
```suggestion
        .assertElementNotPresent('//div[@class="uk-active"]')
```
Also please use variable to store the xpath.